### PR TITLE
Created Menu implementation

### PR DESCRIPTION
**Pull request by AAron122119**
Added a title screen and an empty options menu

### CODE REVIEW

**Comment by kaiiboraka** [comment]
Your menu is BORKED! we need a new option button!

**Comment by AAron122119** [comment]
It is NOT!! You're BORKED!!!

File: src/Assets/Scripts/MainMenu.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenu : MonoBehaviour
+{
+    public void PlayGame()
+    {
+        SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex + 1);
```
Comment (kaiiboraka) [inline src/Assets/Scripts/MainMenu.cs:10]: Consider using scene names instead of indeces.

**Review by kaiiboraka — CHANGES_REQUESTED** [review]
Your main menu needs some updating.

**Review by kaiiboraka — APPROVED** [review]
good stuff mang

**Pull request merged**